crates/interledger-stream/src/client.rs

```diff
@@ -613,11 +610,12 @@ where
                     // Other Rxx errors such as timeouts are likely terminal
                     (_, IlpErrorCode::R01_INSUFFICIENT_SOURCE_AMOUNT) => Ok(()),
                     // Any other error will stop the rest of the payment
-                    _ => Err(Error::SendMoneyError(format!(
-                        "Packet was rejected with error: {} {}",
+                    _ => Err(Error::SendMoneyPacket(
                         reject.code(),
-                        str::from_utf8(reject.message()).unwrap_or_default(),
-                    ))),
+                        str::from_utf8(reject.message())
+                            .unwrap_or_default()
+                            .to_owned(),
```

Comment:
using `String::from_utf8_lossy(reject.message()).to_owned()` might be better because the `unwrap_or_default()` will default to empty `&'static str`.
```suggestion
                        String::from_utf8_lossy(reject.message())
                            .into_owned(),
```

Reply:
However with the lossy it might be better to format with the `{:?}` since it will escape any control chars... Did this one have a test?

Reply:
Yes, it does test but only `assert!(result.is_err())` for unexpected reject 
https://github.com/interledger-rs/interledger-rs/pull/722/files#diff-cdf957aff976c708dad406b806a72a2820bfe77c49895c064783bbb117d202fcR773

Reply:
All right, looks good! Resolving.